apps/demo/story-config/Box.config.tsx

```diff
@@ -37,7 +37,7 @@ export const BoxConfiguration: DemoConfiguration = {
   demo: BoxDemo,
   demoOptions: {},
   stories: {
-    FlexBox: {render: FlexBox},
+      FlexBox: {render: FlexBox},
```

Comment:
This is super not a big deal but random space here

Reply:
Lint agreed with you, idk how they wound up there